### ISSUE TITLE
Recurrent abscesses

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6196]
**Preferred term label:**
Recurrent abscesses

**Synonyms**
Frequent abscesses

**Definition (free text, please give PubMed ID)**
Increased susceptibility to abscesses.
DOI: 10.1530/endoabs.70.EP345

**Parent term (use hpo.jax.org/app)**
Abscess

**Comment by pnrobinson** [comment]
Please use the modifier term Recurrent (https://hpo.jax.org/app/browse/term/HP:0031796).
Otherwise we would need to add Recurrent X abscess etc for all children of Abscess.
